frame/election-provider-multi-phase/src/lib.rs

```diff
@@ -627,6 +627,10 @@ pub mod pallet {
 		#[pallet::constant]
 		type SignedRewardBase: Get<BalanceOf<Self>>;
 
+		// Base reward for a challenger
+		#[pallet::constant]
+		type ChallengeRewardBase: Get<BalanceOf<Self>>;
```

Comment:
The reward of the challenger should be the deposit of the challengee. @emostov wdyt?

Reply:
The reward needs to be less than the deposit so we don't encourage users to spam the system by submitting a bogus solution solution and then challenging their own solution and claiming back their deposit. 

To achieve this we can have a config that is something like `ChallengeDepositDiff`, e.g. if the deposit is 100 and `ChallengeDepositDiff` is 5, then reward base would be 95

Reply:
Yeah, good point. The reward can just be a `Perbill` of the deposit then? I think this is slightly more intuitive than a diff.